File: Go/1043 - Triângulo.go
package main

import (
	"fmt"
)

func main(){

	var a, b, c, peri, area float32

	fmt.Scan(&a, &b, &c)

	peri = a+b+c
	area = ((a+b)*c)/2

	if (b-c) < a  && a < (b+c) {
		fmt.Printf("Perimetro = %.1f\n", peri)
	} else {
		fmt.Printf("Area = %.1f\n", area)
	}
}

File: Ruby/1043 - Triângulo.rb
info = gets.split(' ')

a = info[0].to_f
b = info[1].to_f
c = info[2].to_f

if (b-c) < a  and a < (b+c) then
	puts "Perimetro = #{format("%.1f", a+b+c)}"
else
	puts "Area = #{format("%.1f", ((a+b)*c)/2)}"
end
